requirements.txt

```diff
@@ -30,6 +30,7 @@ django-storages==1.14.2
 django-tables2==2.7.0
 django-tinymce==3.7.1
 django-two-factor-auth==1.16.0
+django-user-agents==0.4.0
```

Comment:
Not sure but couldn't find any other better way yet. May be we can remove this with updated UI for mobile?
For now, we need this to avoid rendering same HTML twice, that'll cause conflict with alpine js dialog boxes.